src/latex/raw.ml

```diff
@@ -206,3 +206,13 @@ let small_table pp ppf tbl =
   break ppf Line
 
 let ocamltag tag pp ppf x = create2 "ocamltag" Fmt.string pp ppf tag x
+
+let math ppf x = math x ppf
```

Comment:
`math` is for rendering a single element, not a block of latex. It adds a `\` prefix that we don't want.

Reply:
fixed